UPtables/script/timetabling.lp

```diff
@@ -52,6 +52,10 @@ N { assigned(C,D,P) : d(D), ppd(P) } N :- course(C,_,N,_,_,_).
 
 :- not { assigned(C,D,P) : c(C) } N, d(D), ppd(P), rooms(N).
 
+%%% Room Unavailability: If a room is unavailable during a certain time, no
+%%% lecture be given in this room at that time.
+:- assigned(C,R,D,P), room_unavailable(R,D,P).
+
```

Comment:
Jes, in the original encoding there are no unavailability constraint for rooms. We will have to decide in the future how to  distinguish between different unavailabilities (room, teacher, ...). We may use something like 
"unavailability(room("H-01"), ..., ....)" or just different predicates for each type of unavailability.